vsock_sample/py/vsock-sample.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+#!/usr/local/bin/env python3
+import argparse
+import socket
+import sys
+
+# Client
+class VsockStream:
+    def __init__(self, conn_tmo=5):
+        self.conn_tmo = conn_tmo
+
+    # Connect to the remote endpoint
+    def connect(self, endpoint):
+        self.sock = socket.socket(socket.AF_VSOCK, socket.SOCK_STREAM)
+        self.sock.settimeout(self.conn_tmo)
+        self.sock.connect(endpoint)
+
+    # Send data to the remote endpoint
+    def send_data(self, data):
+        self.sock.sendall(data)
+        self.sock.close()
+
+def client_handler(args):
+    client = VsockStream()
+    endpoint = (args.cid, args.port)
+    client.connect(endpoint)
+    msg = 'Hello, world!'
+    client.send_data(msg.encode())
+
+# Server
+class VsockListener:
+    def __init__(self, conn_backlog=128):
+        self.conn_backlog = conn_backlog
+    # Bind and listen for connections on the specified port
+    def bind(self, port):
+        self.sock = socket.socket(socket.AF_VSOCK, socket.SOCK_STREAM)
+        self.sock.bind((socket.VMADDR_CID_ANY, port))
+        self.sock.listen(self.conn_backlog)
+    # Receive data from a remote endpoint
+    def recv_data(self):
+        while True:
+            (from_client, (remote_cid, remote_port)) = self.sock.accept()
+            data = from_client.recv(1024).decode()
+            if not data:
+                break
+            print(data)
```

Comment:
I suggest using `print(data, end='')` instead, to not add additional empty lines between buffers.